packages/DataTable/src/DataTable.tsx

```diff
@@ -51,11 +49,11 @@ export interface DataTableProps {
   /** Array of data to be stored in the table */
   data: TableDataItemType[];
 
-  /** The height of the table in px */
-  height?: number;
+  /** The max height of the table in px/vh/calc(xvh+/-xpx) */
+  height?: string;
 
-  /** The width of the table */
-  width?: string | number;
+  /** The max width of the table in px/vw/calc(xvw+/-xpx) */
+  width?: string;
```

Comment:
Since we are breaking the API anyway, maybe we should rename these to `maxHeight` and `maxWidth` to be more explicit and understandable.

Reply:
Sure, I don't have a strong opinion or objection to it 😄